4-WebApp-your-API/4-2-B2C/TodoListService/Startup.cs

```diff
@@ -48,7 +48,7 @@ public void ConfigureServices(IServiceCollection services)
             {
                 // Create policy to check for the scope 'read'
                 options.AddPolicy("ReadScope",
```

Comment:
Why do we use a policy, and not just the RequiredScope attribute? See https://github.com/AzureAD/microsoft-identity-web/wiki/web-apis#using-the-requiredscopes-attribute

Reply:
@jmprieur, I'm not aware about motivation of using policy here but if required I can update it to use RequiredScope attribute. Please let me know if we should update code and Readme for that?

Reply:
> LGTM
> But https://fabrikamb2c.onmicrosoft.com/tasks/access_as_user and https://fabrikamb2c.onmicrosoft.com/tasks/read both exist. What problem do we solve with this PR, @Shama-K ?

This PR makes pre-registered config details consistent with Readme steps. Readme steps to register the API adds "access_as_user" scope, whereas config file had "read" as scope value.

Reply:
Yes, I think we should align with what we recommend in Microsoft.Identity.Web, Shama

![image](https://user-images.githubusercontent.com/13203188/111600919-234c0a80-87d2-11eb-8086-c65c5f7fbd30.png)

Reply:
@jmprieur, updated the code as per suggestion.